frontend/src/pages/projects/screens/detail/storage/StorageTable.tsx

```diff
@@ -15,20 +20,65 @@ type StorageTableProps = {
 const StorageTable: React.FC<StorageTableProps> = ({ pvcs, refresh, onAddPVC }) => {
   const [deleteStorage, setDeleteStorage] = React.useState<PersistentVolumeClaimKind | undefined>();
   const [editPVC, setEditPVC] = React.useState<PersistentVolumeClaimKind | undefined>();
+  const isStorageClassesAvailable = useIsAreaAvailable(SupportedArea.STORAGE_CLASSES).status;
+  const [storageClasses, storageClassesLoaded] = useStorageClasses();
+  const [alertDismissed, setAlertDismissed] = React.useState<boolean>(false);
+  const storageTableData: StorageTableData[] = pvcs.map((pvc) => ({
+    pvc,
+    storageClass: storageClasses.find((sc) => sc.metadata.name === pvc.spec.storageClassName),
```

Comment:
`storageClass` takes undefined value when storageClass is deleted from `openshift` cluster

Reply:
thats fine. so you will use the pvc name as fallback

Reply:
yeah, I have updated the pr according to what was discussed